api/models/User.js

```diff
@@ -97,12 +97,11 @@ module.exports = bookshelf.Model.extend(merge({
   },
 
   posts: function () {
-    return this.belongsToMany(Post).through(PostUser).query(q => q.where(function () {
-      this.where('type', null).orWhere('type', '!=', Post.Type.THREAD)
-    }))
+    return this.hasMany(Post).query(q => q.where('type', '!=', Post.Type.THREAD))
   },
 
   projects: function () {
+    // TODO: fix
```

Comment:
Now or later?

Reply:
later, whenever we get back to the projects functionality in a real way